.gitignore

```diff
@@ -1,7 +1,7 @@
 build
-opt
 js/handlebars.js
 js/lozad.js
 vendor/
 models/1
 clover.xml
+translationtool.phar
```

Comment:
I didn't use this before. Is this "standard" tooling in Nextcloud world? E.g. can we move to transifex with .po files. Asking because I know .po files are very powerful (and you can have multiple plurals, like Serbian language), but Transifex (which hosts most of NC and NC apps) have only 1 plural. Not sure if this is due to 1) Transifex being limited, or 2) other NC apps are using some less powerful format than .po